specification/agrifood/data-plane/Microsoft.AgFoodPlatform/preview/2022-11-01-preview/agfood.json

```diff
@@ -23632,7 +24067,8 @@
             "key2": 123.45
           }
         }
-      }
+      },
+      "additionalProperties": {}
```

Comment:
I am not sure the purpose of this "additionalProperties"

Reply:
Removed.